assets/js/content.js

```diff
@@ -101,8 +101,10 @@ const myTimeout = setTimeout(function(){
         }
 
         if(settings.cheers){
-          $("#"+art_id+" a.post-award-btn").hide();
-          $("#"+art_id+" post-award-users").hide();
+          // Hide 'Cheered by' in the post header
+          $('.post-award-users').hide();
+          // Hide Cheers button
+          $('*[data-v-361701b8=""]').hide();
```

Comment:
the string 'data-v-361701b8' is randomly generated when the page is built, every cheer button will have a different random data attribute